lib/trln_argon/controller_override.rb

```diff
@@ -100,11 +100,15 @@ module ControllerOverride
         # Facets to be populated on landing page
         config.add_home_facet_field TrlnArgon::Fields::ACCESS_TYPE_FACET.to_s,
                                     label: TrlnArgon::Fields::ACCESS_TYPE_FACET.label,
-                                    collapse: false
+                                    collapse: false,
+                                    show: true,
+                                    partial: 'catalog/facet_checkbox',
+                                    locals: { checkbox_field: 'Online', checkbox_field_label: 'Available Online' }
```

Comment:
@mikedaul could this be in a translation instead of being hard-coded in the controller?

Reply:
@corylown If I'm understanding what you're asking, placing the facet field and/or label in the translation makes it more complicated to have more than one checkbox facet active at a time. I think passing them as locals makes things a little easier to manage, but I'm open to other options :)

Reply:
Can you call the translation from within the locals in the controller?

`locals: { checkbox_field: 'Online', checkbox_field_label: i18n('path.to.translation') }`

Reply:
That's a great idea! I'll give it a try.